### ISSUE TITLE
Please lock com.google.android.gms:play-services-ads-identifier:16+ to 17.0.0

### ISSUE THREAD

**Issue by hamberluo** [RodrigoSMarques/flutter_branch_sdk#97]
enhance build time

**Comment by RodrigoSMarques** [comment]
I will evaluate.

There are recommendations before using this version.

[https://help.branch.io/developers-hub/docs/android-basic-integration](https://help.branch.io/developers-hub/docs/android-basic-integration)

![Captura de Tela 2021-04-12 às 09 42 30](https://user-images.githubusercontent.com/17687286/114396026-71ef8900-9b73-11eb-89c5-b4e0edfd7d43.png)

**Comment by hamberluo** [comment]
thanks
i think using a specific version is better than a fuzzy version

**Comment by guilhermeagostinelli** [comment]
It's worth mentioning that I'm unable to build in release mode today due to this 16+ version. Please see more details in this reopened issue: https://github.com/RodrigoSMarques/flutter_branch_sdk/issues/84

**Comment by RodrigoSMarques** [comment]
No errors here. I just compiled the project. See the time.

![Captura de Tela 2021-05-12 às 18 15 28](https://user-images.githubusercontent.com/17687286/118045294-1bca5d00-b34e-11eb-87b4-8e7f98ddb81e.png)

Branch's documentation mentions a series of notes for using version 17 and this can impact other projects.

![Captura de Tela 2021-05-12 às 18 20 16](https://user-images.githubusercontent.com/17687286/118045777-c0e53580-b34e-11eb-8aca-5f08d4a3b890.png)

**Comment by guilhermeagostinelli** [comment]
Hi @RodrigoSMarques , thanks for the reply. The problem is gone for me too. It seems that there's some instability with the servers that serve the play-services-ads-identifier:16+ lib

Yesterday during almost the whole day I had to change implementation 'com.google.android.gms:play-services-ads-identifier:16+' to implementation 'com.google.android.gms:play-services-ads-identifier:17.0.0' in my app/build.gradle file otherwise I couldn't even execute the project in debug mode.

Then, when trying to run flutter run --release, I got the error message mentioned in https://github.com/RodrigoSMarques/flutter_branch_sdk/issues/84.

Today the server that serves the play-services-ads-identifier:16+ lib seems to be back online and I'm able to debug and build the app again independently of using 16+ or 17.0.0 in my app/build.gradle file.
